static/js/brand-store/NoStores/NoStores.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import NoStores from "./NoStores";
+
+test("the 'is public' checkbox should not be checked when the current store is set to private", () => {
```

Comment:
There is no checkbox in this component?

Reply:
sorry - doing too many things at once! fixed